test/cmd/images-old-policy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+project=$(oc project -q)
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc login -u system:admin
+  oc project ${project}
+  oadm policy reconcile-cluster-roles --additive-only=false --confirm
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/images-old-policy"
+
+os::cmd::expect_success 'oadm policy add-role-to-user admin image-user -n ${project}'
```

Comment:
The rule is fairly simple until you start embedding JSONPath stuff or templates:
 - surround the command you're passing through with double quotes
 - if you're expanding variables in that command, surround *that* with single quotes

```
os::cmd::expect_success "oadm policy add-role-to-user admin image-user -n '${project}'"
```

This gets us:
 - `os::cmd` failures that show you *exactly* what ran with all the var substituted, especially if you're doing a hot-loop test locally and not tearing things down this allows you to copy-paste and re-run stuff yourself
 - proper quoting so things don't break down the line if your vars contain white space